assets/src/room.ts

```diff
@@ -105,6 +114,15 @@ export class Room {
             });
           }
           this.tracks.get(ctx.peer.id)?.push(ctx);
+
+          if (ctx.track?.kind === "audio") {
+            setVisibilityOfMuteIndicator(ctx.peer.id, ctx.metadata.active);
+          } else if (
+            ctx.track?.kind === "video" &&
+            !(ctx.metadata && ctx.metadata.type === "screensharing")
```

Comment:
```suggestion
            ctx.track?.kind === "video" && ctx.metadata?.type !== "screensharing"
```
You may also change the condition in line 104.